src/services/decoders/ClpIrDecoder/index.ts

```diff
@@ -130,7 +149,7 @@ class ClpIrDecoder implements Decoder {
                 console.error("Formatter is not set for structured logs.");
```

Comment:
Is this case possible? If not, why not throw an error? If it is possible, then in the structured case, we should continue returning the `results` unchanged, right?

Reply:
Right, we should throw an error here. Until we start using formatters for unstrcutured logs as well